packages/E2ETest/wdio/test/testInput.spec.ts

```diff
@@ -26,6 +26,16 @@ describe('First', () => {
   it('Type abc on TextInput', () => {
     TextInputTestPage.clearAndTypeOnTextInput('abc');
     assert.equal(TextInputTestPage.getTextInputText(), 'abc');
+
+    // We should really develop a proper mechanism of waiting for idle, or detecting when the JS + bridge has had a chance of completing
+    // But this test is causing a large itermittent failure rate on CI, so adding this polling timeout for now.
+    // eslint-disable-next-line no-undef
+    browser.waitUntil(
+      () => TextInputTestPage.getTextInputPrev2Text().includes('onKeyPress'),
+      1000,
```

Comment:
if this is all it takes, great, but do we not need to wait in other parts of the test? IIRC I was seeing failures happening afterwards, when sending keystrokes. Maybe we need to add a delay on some of the test utility functions instead?

Reply:
Lets try this for now and see how much it helps.  I consider this a bit of a hack, and would rather not apply it broadly.